PennMobile/Login/LoginController.swift

```diff
@@ -48,12 +48,20 @@ class LoginController: UIViewController, ShowsAlert {
 }
 
 // MARK: - Login Completion Handler
-extension LoginController {
+extension LoginController: NotificationRequestable {
     func loginCompletion(_ successful: Bool) {
         if successful {
             // Login Successful
             UserDefaults.standard.setLastLogin()
             AppDelegate.shared.rootViewController.switchToMainScreen()
+            
+            DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
```

Comment:
We should do this on a non-main thread. If the Notification Center stuff wants to happen on a main thread, we can put that inside `self.requestNotification()`

Reply:
What exactly are you suggesting? This is just a 0.5 second delay. Notification Center stuff still happens on the background thread.

Reply:
All this is saying is asynchronously wait 0.5 seconds and then from the Main thread, call requestNotification(). Request notification then asks Notification Center if a notification has been enabled, which then responds on a background thread "yes" or "no."

Reply:
just suggesting the work gets offloaded to a background thread, since it's not UI based or user focused.

so that would be `DispatchQueue.global(qos: .background).asyncAfter(deadline: .now() + 0.5)`

Reply:
main queue handles UI and keeps things 60fps, so anytime we can async on a background queue we should

Reply:
Implemented.